install_config/topics/configuring_for_azure_storage.adoc

```diff
@@ -0,0 +1,54 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-storage_{context}']
+= Configuring {product-title} to use Microsoft Azure storage
+
+{product-title} can use Microsoft Azure storage using persistent volumes
+mechanisms. {product-title} creates the disk in the resource group and attaches
+the disk to the correct instance.
+
+.Procedure
+
+. The following `storageclass` is created when you configure the Azure cloud
+provider at installation using the `openshift_cloudprovider_kind=azure` and
+`openshift_cloud_provider_azure*` variables in the Ansible inventory:
++
+[source,bash]
+----
+$ oc get --export storageclass azure-standard -o yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.beta.kubernetes.io/is-default-class: "true"
+  creationTimestamp: null
+  name: azure-standard
+parameters:
+  kind: Shared
+  storageaccounttype: Standard_LRS
+provisioner: kubernetes.io/azure-disk
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+----
++
+If you did not use Ansible to enable {product-title} and Microsoft Azure integration, you can create the `storageclass` manually. See the xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#[Dynamic provisioning and creating storage classes section] for more information.
+
+. Currently, the default `storageclass` kind is `shared` which means that the
+Microsoft Azure instances must use unmanaged disks. You can optionally modify
+this by allowing instances to use managed disks by providing the
+`openshift_storageclass_parameters={'kind': 'managed', 'storageaccounttype':
```

Comment:
Here the kind of  `managed` should be `Managed` with capital `M`, otherwise not work.